sources/Valkey.Glide/Commands/Options/HashFieldExpirationConditionOptions.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+namespace Valkey.Glide.Commands.Options;
+
+/// <summary>
+/// Optional arguments for hash field expiration commands (HEXPIRE, HPEXPIRE, HEXPIREAT, HPEXPIREAT).
+/// </summary>
+/// <remarks>
+/// See <a href="https://valkey.io/commands/hexpire/">valkey.io</a>
+/// </remarks>
+public class HashFieldExpirationConditionOptions
+{
+    /// <summary>
+    /// The expiration condition.
+    /// </summary>
+    public ExpireOptions? Condition { get; set; }
+
+    /// <summary>
+    /// Sets the expiration condition.
+    /// </summary>
+    /// <param name="condition">The expiration condition</param>
+    /// <returns>This HashFieldExpirationConditionOptions instance for method chaining</returns>
+    public HashFieldExpirationConditionOptions SetCondition(ExpireOptions condition)
+    {
+        Condition = condition;
+        return this;
+    }
+}
+
+/// <summary>
+/// Expiration condition options for hash field expiration commands.
+/// </summary>
+public enum ExpireOptions
+{
+    /// <summary>
+    /// Set expiration only if the field has no expiration.
+    /// </summary>
+    HAS_NO_EXPIRY,
```

Comment:
Enums in C# usually use upper camel case (and a quick check of other enums in this repo shows the same).